definitions/variable/emissions/emissions.yaml

```
@@ -1,94 +1,179 @@
 # Emissions species are grouped into three levels of detail for the reporting.
 # Level-1 reports aggregate emissions, Level-3 has the most detailed disaggregation.
 # Gross emissions are only defined for CO2.
+# Emissions reporting guidance follows IPCC guidance, referring to 
+# 2006 categories: https://www.ipcc-nggip.iges.or.jp/public/2019rf/pdf/1_Volume1/19R_V1_Ch08_Reporting_Guidance.pdf
+# 1996 categories: https://github.com/primap-community/climate_categories/blob/main/climate_categories/data/IPCC1996.yaml
 
 - Emissions|{Level-1 Species}:
-    description: Emissions of {Level-1 Species}
+    description: Emissions of {Level-1 Species} to the atmosphere. 
+      For the avoidance of doubt, these emissions are net emissions 
+      (i.e. include both positive (emission) and negative (removal) 
+      contributions. For positive contributions only, see 
+      "Gross Emissions|*").
     unit: "{Level-1 Species}"
+    notes: For a breakdown of carbon removals, see the "Carbon Removal|*" tree
     components:
       - Emissions|{Level-1 Species}|AFOLU
       - Emissions|{Level-1 Species}|Energy
       - Emissions|{Level-1 Species}|Industrial Processes
       - Emissions|{Level-1 Species}|Waste
-      - Emissions|{Level-1 Species}|Capture and Removal
+      - Emissions|{Level-1 Species}|Other Capture and Removal
+      - Emissions|{Level-1 Species}|Product Use
       - Emissions|{Level-1 Species}|Other
 - Gross Emissions|CO2:
-    description: Gross emissions of carbon dioxide (CO2), not accounting for negative
-      emissions from bioenergy with CCS (BECCS) or agriculture, forestry
-      and other land use (AFOLU)
+    description:  Gross emissions of carbon dioxide (CO2), not accounting for
+      negative emissions, for example from bioenergy with CCS (BECCS), direct air carbon capture
+      or agriculture, forestry and other land use (AFOLU)
+    notes: For net emissions (accounting for negative emissions), see "Emissions|CO2".
     unit: Mt CO2/yr
 
 - Emissions|{Level-2 Species}|AFOLU:
     description: Emissions of {Level-2 Species} from agriculture, forestry
-      and other land use (IPCC category 3)
+      and other land use (IPCC 1996 category 4, 5; IPCC 2006 category 3).
+      In particular, this includes agricultural emissions (mostly non-CO2) 
+      from livestock, rice cultivation and soil management as well as 
+      emissions from land use and land-use change (mostly CO2) such as deforestation, 
+      conversion of non-forest natural land, and drained peatlands. 
+      This is reporting the sum of sources and sinks including land-use methods that do not 
+      use CCS such as soil carbon sequestration, biochar, forestry, and re/afforestation and 
+      agroforestry. 
```

```suggestion
      This reports the sum of sources and sinks including land-use methods that do not 
      use CCS such as soil carbon sequestration, biochar, forestry, and re/afforestation and 
      agroforestry. 
```

My immediate question here would be, where does stuff that does use CCS go? Not sure if that's worth clarifying or not (the notes already clarifies a bit, but not unambiguously).

---

Do you have examples of CCS on AFOLU?

---

> Do you have examples of CCS on AFOLU where it would not be clear that it should go under this variable?

I meant more the other way around. If I have a variable with CCS, I would not know where to put it (I can go looking through all the other categories, but we can also just say, "Look at the energy tree").

> Or should we just delete "that do not use"

Would also be fine

---

OK I did the latter in c4d4cc91c2f29750c1a0e6aeed7d8dd375884480